Automattic/jetpack
# Duplicated views: add untangled comments to include list

Fixes:
- https://github.com/Automattic/wp-calypso/issues/95622

## Proposed changes:

* Added the comments page to the list of duplicated view pages.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply these changes to your sandbox.
* If not merged, use the Calypso live link found in this [PR](https://github.com/Automattic/wp-calypso/pull/97559)
* Enable the experiment treatment group (22129-explat-experiment)
* Go to the comments section
* You should see the untangled version with no view selector.
